extensions/amp-story/0.1/related-articles.js

```diff
@@ -56,11 +56,20 @@ function buildArticleFromJson_(articleJson) {
 
   user().assert(isProtocolValid(articleJson['url']),
       `Unsupported protocol for article URL ${articleJson['url']}`);
+  dev().assert(articleJson['url']);
+
+  let domain;
+  try {
+    domain = getSourceOrigin(articleJson['url']);
+  }
+  catch (err) {
```

Comment:
Nit: `} catch() {`
It should be on the same line, and `err` is unused

Reply:
leaving `catch()` without args causes lint error